hardware/gyros.py

```diff
@@ -162,6 +188,7 @@ def get_scale_factor(self, sensitivity=None, velocity=1, pitch=None):
         lia.time_constant = 0.01
 
         # set the rotation speed and store the current
+        # keep

```

Comment:
There's a couple of stray "keeps" in your comments. Let's clean those up if possible.